src/ImageSharp/Configuration.cs

```diff
@@ -173,6 +174,17 @@ public Configuration Clone()
             };
         }
 
+        /// <summary>
+        /// Registers the webp format detector, encoder and decoder.
+        /// </summary>
+        public void AddWebp()
```

Comment:
I think these should be per format extension methods.

Reply:
Yes, definitely. It should live in the `Experimental` namespace.

Reply:
Also: I'm not sure how useful our `IConfigurationModule` abstraction is, but since it's the current practice, we need to implement it for WebP:

https://github.com/SixLabors/ImageSharp/blob/78a584e8482b052d7a9885682299e2f37518d83d/src/ImageSharp/Formats/Jpeg/JpegConfigurationModule.cs

Reply:
yes makes sense, i have moved it to a extension method. 

The WebpConfigurationModule.cs already exists: [here](https://github.com/SixLabors/ImageSharp/blob/webp/src/ImageSharp/Formats/WebP/WebpConfigurationModule.cs)